internal/bminventory/inventory.go

```diff
@@ -2180,6 +2181,16 @@ func (b *bareMetalInventory) GetPresignedForClusterFiles(ctx context.Context, pa
 	var err error
 	fullFileName := fmt.Sprintf("%s/%s", params.ClusterID, params.FileName)
 	downloadFilename := params.FileName
+	if params.FileName == "manifests" {
```

Comment:
@avishayt how can this function be tested ? When I'm trying to run it against test-infra env, it fails since test infra using scality and not aws.

```
→ curl -X GET "${CLUSTER_API_URL}/downloads/files-presigned?file_name=manifests&additional_name=99_openshift-lso-olm3.yaml" -H "accept: application/json"
{"code":"400","href":"","id":400,"kind":"Error","reason":"Failed to generate presigned URL: invalid backend"}
```

Reply:
You can test by:
1. Deploy scality such that it's exposed to users
2. Change IsAwsS3() method in s3wrapper/client.go to always return true

Reply:
Thanks, verified:
```
→ curl -s  -X GET "${CLUSTER_API_URL}/downloads/files-presigned?file_name=manifests&additional_name=manifests/99_openshift-localvolume_crd.yaml" -H "accept: application/json" | jq
{
  "url": "http://cloudserver-front:8000/test/828315fb-9034-4289-a398-c9465e8e8ef9/manifests/manifests/99_openshift-localvolume_crd.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=accessKey1%2F20201014%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20201014T112234Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3Bfilename%3D99_openshift-localvolume_crd.yaml&X-Amz-Signature=f36cbda4ee7b09984fca42bec35f239a0978b7ec64af7f769fe549ad74d4f7c2"                         
}

→ curl 'http://cloudserver-front:8000/test/828315fb-9034-4289-a398-c9465e8e8ef9/manifests/manifests/99_openshift-localvolume_crd.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=accessKey1%2F20201014%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20201014T112234Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3Bfilename%3D99_openshift-localvolume_crd.yaml&X-Amz-Signature=f36cbda4ee7b09984fca42bec35f239a0978b7ec64af7f769fe549ad74d4f7c2'
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  name: localvolumes.local.storage.openshift.io
spec:
  group: local.storage.openshift.io
  names:
...
```